packages/@sfx/sayt-driver-plugin/test/unit/common/sayt-driver-plugin.test.ts

```diff
@@ -222,40 +236,44 @@ describe('Sayt Driver Plugin', () => {
 
   describe('fetchSaytData()', () => {
     let dispatchEvent;
-    let fetchEvent;
-    let response;
+    let results;
+    let searchbox;
     let sendSaytApiRequest;
 
     beforeEach(() => {
       dispatchEvent = dom_events.dispatchEvent = spy();
       driver.core = {
         dom_events,
       };
-      fetchEvent = {
-        detail: {
-          query: 'shirt',
-        }
-      };
-      response = { a: 'b' };
+      results =  { a: "b" };
       sendSaytApiRequest = stub(driver, 'sendSaytApiRequest');
     });
 
-    it('should get a response from Sayt client request method', () => {
-      const saytPayload  = fetchEvent.detail;
-      sendSaytApiRequest.resolves(response);
+    it('should call sendSaytApiRequest with query from event and undefined config', () => {
+      const saytQueryPayload  = fetchEvent.detail;
+      const undefinedConfig = { config: undefined }
```

Comment:
Generally, instead of setting things to `undefined`, you often can just leave it out.